Description: add format argument to printf, otherwise we get an error with -Werror=format-security
Origin: vendor
Bug-Debian: http://bugs.debian.org/643452
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-12-21

--- a/Subpoly.c
+++ b/Subpoly.c
@@ -867,7 +867,7 @@
 
   if(rFlag==0) Write_List_2_File(polyo,_NFL); 
   _NFL->TIME=time(NULL); 
-  printf(ctime(&_NFL->TIME));
+  printf("%s", ctime(&_NFL->TIME));
   free(_NFL);
 }
 
@@ -1190,7 +1190,7 @@
     
   printf("max_order=%d\n", max_order);
   Write_List_2_File(polyo,_NFL); 
-  _NFL->TIME=time(NULL); printf(ctime(&_NFL->TIME));
+  _NFL->TIME=time(NULL); printf("%s", ctime(&_NFL->TIME));
   free(_NFL);
 }
 
--- a/Subadd.c
+++ b/Subadd.c
@@ -95,7 +95,7 @@
 NF_List *AuxNFLptr=NULL;		/* dirty trick for Xmin Xmax Xdif */
 void Init_NF_List(NF_List *L)
 {    
-     L->TIME= L->SAVE= time(NULL); printf(ctime(&L->TIME)); L->CLOCK= clock();
+     L->TIME= L->SAVE= time(NULL); printf("%s", ctime(&L->TIME)); L->CLOCK= clock();
      L->IP_Time = L->NF_Time = 0;  L->d = L->nNF = L->nIP = L->nSLNF = 0;
      
      Init_FInfoList(&L->In);	if(*L->iname) Read_In_File(L);	
--- a/Subdb.c
+++ b/Subdb.c
@@ -1762,7 +1762,7 @@
 	}
      }	if(*dbin) Close_DB(DB);
      printf("max_order=%d\n", max_order); Write_List_2_File(polyo,_L); 
-     _L->TIME=time(NULL); printf(ctime(&_L->TIME));
+     _L->TIME=time(NULL); printf("%s", ctime(&_L->TIME));
      free(_L);
 }
 void VPHM_Sublat_Polys(char sFlag,char mr,char *dbin,char *polyi,char *polyo, 
